File: connecttoapilibrary/src/main/java/com/sec/connecttoapilibrary/onlinefm/OnLineFMPlayerListener/ObserverPlayerListener.java
package com.sec.connecttoapilibrary.onlinefm.OnLineFMPlayerListener;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/2/7.
 */

public interface ObserverPlayerListener {
    void observerStartPlayer(String mPlayer_Url, int play_type);
    void observerStartDNS(boolean isStart);
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/OnLineFMPlayerListener/ObserverPlayerListener.kt
package com.sec.myonlinefm.OnLineFMPlayerListener

/**
 * Created by SRC-TJ-MM-BinYang on 2018/2/7.
 */
interface ObserverPlayerListener {
    open fun observerStartPlayer(mPlayer_Url: String?, play_type: Int)
    open fun observerStartDNS(isStart: Boolean)
}